src/urllib3/contrib/ntlmpool.py

```diff
@@ -20,6 +32,15 @@ class NTLMConnectionPool(HTTPSConnectionPool):
     Implements an NTLM authentication version of an urllib3 connection pool
     """
 
+    warnings.warn(
```

Comment:
You can remove this warning, everything is covered by the one above.

Reply:
Okey